src/commands/bitfield.json

```diff
@@ -44,84 +44,91 @@
                 "key_spec_index": 0
             },
             {
-                "token": "GET",
-                "name": "encoding_offset",
-                "type": "block",
-                "optional": true,
-                "arguments": [
-                    {
-                        "name": "encoding",
-                        "type": "string"
-                    },
-                    {
-                        "name": "offset",
-                        "type": "integer"
-                    }
-                ]
-            },
-            {
-                "token": "SET",
-                "name": "encoding_offset_value",
-                "type": "block",
-                "optional": true,
-                "arguments": [
-                    {
-                        "name": "encoding",
-                        "type": "string"
-                    },
-                    {
-                        "name": "offset",
-                        "type": "integer"
-                    },
-                    {
-                        "name": "value",
-                        "type": "integer"
-                    }
-                ]
-            },
-            {
-                "token": "INCRBY",
-                "name": "encoding_offset_increment",
-                "type": "block",
-                "optional": true,
+                "name": "operation",
+                "type": "oneof",
+                "multiple": "true",
                 "arguments": [
                     {
-                        "name": "encoding",
-                        "type": "string"
-                    },
-                    {
-                        "name": "offset",
-                        "type": "integer"
+                        "token": "GET",
+                        "name": "encoding_offset",
+                        "type": "block",
+                        "optional": true,
```

Comment:
since these are now part of `oneof`, is the `optional` still valid?

Reply:
Correct, the optional needs to be removed. Resolved in 56613896e